OpenUtau.Core/Api/G2pDictionary.cs

```diff
@@ -30,13 +30,20 @@ class TrieNode {
                         .Where(parts => parts.Length == 2)
                         .ToList()
                         .ForEach(parts => builder.AddSymbol(parts[0], parts[1]));
-                    Core.Api.Resources.cmudict_0_7b.Split('\n')
-                        .Where(line => !line.StartsWith(";;;"))
-                        .Select(line => line.Trim().ToLowerInvariant())
-                        .Select(line => line.Split(new string[] { "  " }, StringSplitOptions.None))
-                        .Where(parts => parts.Length == 2)
-                        .ToList()
-                        .ForEach(parts => builder.AddEntry(parts[0], parts[1].Split().Select(symbol => RemoveTailDigits(symbol))));
+                    Core.Api.Resources.cmudict_0_7b.Split('\n').AsParallel().ForAll(line => {
```

Comment:
Parallelizing a lot of small tasks is overkill and may actually slow things down.
A single thread as the original is not too bad after you made it async. The builder is not thread-safe either.